### ISSUE TITLE
The learning_rate of classifier

### ISSUE THREAD

**Issue by Alphia** [lfz/DSB2017#118]
The training of classifier needs a default learning rate, which is 'None' in the code, what value should I use as default?

Thanks a lot!

**Comment by Alphia** [comment]
My-trained-detector works well with your pre-trained classifier.
My-trained-detector works with my-trained-classifier, got log loss 23.

I set the learning rate of the classifier as 0.01. Am I on the wrong way? @lfz
